### PR TITLE
done hw 13

### PR DESCRIPTION
**Pull request by sergvetrov**
dont make tests in AnimalService methods, because this class using methods from another classes who already tested

### CODE REVIEW

File: src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -1,6 +1,66 @@
...
     public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        AnimalService service = new AnimalService();
+        System.out.println("Hello in pet shelter, select one function: ");
+        System.out.println("1. Add pet (add);");
+        System.out.println("2. Show all pets (show);");
+        System.out.println("3. Take pet from shelter (take);");
+        System.out.println("3. Exit (exit);");
+
+        while (true) {
```
Comment (shaitanaleksander) [inline src/main/java/org/example/Main.java]: while true need to be avoided

File: src/test/java/org/example/service/AnimalServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    }
+
+    @Test
+    public void testShowAll() {
+    }
+
+    @Test
+    public void testRemove() {
+    }
+}
```
Comment (shaitanaleksander) [inline src/test/java/org/example/service/AnimalServiceTest.java]: why do you need these

**Review by shaitanaleksander — APPROVED** [review]

File: src/main/java/org/example/Main.java (hunk 1 above)
Comment (sergvetrov) [reply src/main/java/org/example/Main.java]: fixed

File: src/test/java/org/example/service/AnimalServiceTest.java (hunk 2 above)
Comment (sergvetrov) [reply src/test/java/org/example/service/AnimalServiceTest.java]: stupid mistake, fixed (deleted)

**Pull request merged**